### ISSUE TITLE
matchesXPath with equalToXml not working

### ISSUE THREAD

**Issue by fabiojb** [tomakehurst/wiremock#753]
A stub mapping like that it's not working:
```
"bodyPatterns": [
            {
                "matchesXPath": {
                    "expression": "//todo-item",
                    "equalToXml": "<todo-item>Do the washing</todo-item>"
                }
            }
        ]
```

But according to the [documentation](http://wiremock.org/docs/request-matching/#nested-value-matching-1), it's supposed to work.

I noticed that when I have a mapping like the above, the wiremock server(standalone) stops logging requests on the `__admin/requests`.

**Comment by tomakehurst** [comment]
Can you elaborate "not working"?

**Comment by fabiojb** [comment]
Yes, the requests that should be mocked by the mapping above are not being intercepted, and wiremock also stop logging any requests on the `__admin/requests` and `__admin/requests/unmatched` urls.

**Comment by tomakehurst** [comment]
Please can you elaborate the following:
1) The full stub mapping you've configured (not just the body matcher)
2) The full request and response you're making
3) The options you started WireMock with

**Comment by fabiojb** [comment]
Stub mapping:
```
{
    "id": "1aa8b881-2e21-3822-beca-9d6923a5c2fe",
    "request": {
        "url": "/ebs/ppf/tcbo/v3/3.1/TCCaptacaoPS?wsdl",
        "method": "POST",
        "bodyPatterns": [
            {
                "matchesXPath": {
                    "expression": "/S:Envelope/S:Body/ns1:listarCaptacao/inListarCaptacao",
                    "equalToXml": "<inListarCaptacao><codigoCanal>1</codigoCanal><codigoLoja>3</codigoLoja><codigoTipoCaptacao>1</codigoTipoCaptacao></inListarCaptacao>"
                },
                "xPathNamespaces": {
                    "S": "http://schemas.xmlsoap.org/soap/envelope/",
                    "ns1": "http://ws.cadastro.totvs.com/v3"
                }
            }
        ]
    },
    "response": {
        "status": 200,
        "bodyFileName": "body-3.1-TCCaptacaoPS-2m29k.xml",
        "headers": {
            "Date": "Tue, 12 Sep 2017 17:09:37 GMT",
            "Content-Type": "text/xml; charset=utf-8"
        }
    },
    "uuid": "1aa8b881-2e21-3822-beca-9d6923a5c2fe"
}
```

Response:
```
<?xml version='1.0' encoding='UTF-8'?>
<S:Envelope xmlns:S="http://schemas.xmlsoap.org/soap/envelope/">
  <S:Header>
    <WorkContext xmlns="http://oracle.com/weblogic/soap/workarea/">rO0ABXe9ACx3ZWJsb2dpYy5hcHAuSUYtdG90dnMtdG90dnNjYXJkLXNlcnZpY29zLWVhcgAAANYAAAAjd2VibG9naWMud29ya2FyZWEuU3RyaW5nV29ya0NvbnRleHQABjMuMTUuMgAld2VibG9naWMuYXBwLnBwZnNlcnZpY2VzLWNhZGFzdHJvLWVhcgAAANYAAAAjd2VibG9naWMud29ya2FyZWEuU3RyaW5nV29ya0NvbnRleHQABjMuMjEuMAAA</WorkContext>
  </S:Header>
  <S:Body>
    <ns2:listarCaptacaoResponse xmlns:ns2="http://ws.cadastro.totvs.com/v3">
      <outListarCaptacao>
        <status>
          <mensagem>Transação realizada com sucesso.</mensagem>
          <versao>3.15.2</versao>
          <codigo>S00001</codigo>
        </status>
        <totalRegistros>1</totalRegistros>
        <captacoes>
          <captacao>
            <codigoCanal>1</codigoCanal>
            <codigoCaptacao>70</codigoCaptacao>
            <codigoLoja>3</codigoLoja>
            <descricaoCanal>LOJA</descricaoCanal>
            <descricaoCaptacao>NORMAL</descricaoCaptacao>
            <descricaoLoja>SHOPPING JOÃO PESSOA</descricaoLoja>
            <identificadorSituacaoCaptacao>A</identificadorSituacaoCaptacao>
            <tipoCaptacao>1</tipoCaptacao>
          </captacao>
        </captacoes>
      </outListarCaptacao>
    </ns2:listarCaptacaoResponse>
  </S:Body>
</S:Envelope>
```

Request:
```
<?xml version='1.0' encoding='UTF-8'?>
<S:Envelope xmlns:S="http://schemas.xmlsoap.org/soap/envelope/">
  <S:Header>
    <ns1:header xmlns:ns1="http://ws.cadastro.totvs.com/v3">
      <loja>3</loja>
      <senhaServico>totvs123</senhaServico>
      <usuario>00115126</usuario>
      <usuarioServico>weblogic</usuarioServico>
    </ns1:header>
    <WorkContext xmlns="http://oracle.com/weblogic/soap/workarea/">rO0ABXe9ACx3ZWJsb2dpYy5hcHAuSUYtdG90dnMtdG90dnNjYXJkLXNlcnZpY29zLWVhcgAAANYAAAAjd2VibG9naWMud29ya2FyZWEuU3RyaW5nV29ya0NvbnRleHQABjMuMTUuMgAld2VibG9naWMuYXBwLnBwZnNlcnZpY2VzLWNhZGFzdHJvLWVhcgAAANYAAAAjd2VibG9naWMud29ya2FyZWEuU3RyaW5nV29ya0NvbnRleHQABjMuMjEuMAAA</WorkContext>
  </S:Header>
  <S:Body>
    <ns1:listarCaptacao xmlns:ns1="http://ws.cadastro.totvs.com/v3">
      <inListarCaptacao>
        <codigoCanal>1</codigoCanal>
        <codigoLoja>3</codigoLoja>
        <codigoTipoCaptacao>1</codigoTipoCaptacao>
      </inListarCaptacao>
    </ns1:listarCaptacao>
  </S:Body>
</S:Envelope>
```
Wiremock CLI:
`java -jar wiremock-standalone-2.7.1.jar --port=8011`

**Comment by tomakehurst** [comment]
The feature you're trying to use was released in 2.8.0. You might want to try upgrading to that.

**Comment by fabiojb** [comment]
It worked on 2.8.0, thanks!
